.expo/settings.json

```diff
@@ -0,0 +1,8 @@
+{
+  "hostType": "lan",
+  "lanType": "ip",
+  "dev": true,
+  "minify": false,
+  "urlRandomness": null,
+  "https": false
+}
```

Comment:
I think this expo file got here by mistake when you were trying to run the project. You should remove it.